.github/workflows/build.yml

```diff
@@ -272,72 +273,72 @@ jobs:
-  #   steps:
-  #     - name: Install host dependencies
-  #       run: |
-  #         sudo apt-get update
-  #         sudo apt-get install protobuf-compiler krb5-user libkrb5-dev libclang-dev liblzma-dev libssl-dev pkg-config
-  #     - name: Checkout
-  #       uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
-  #       with:
-  #         submodules: recursive
-  #     - uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435 # v4
-  #       if: ${{ github.event_name == 'pull_request' }}
-  #     - uses: dtolnay/rust-toolchain@1.68.2
-  #       with:
-  #         components: rustfmt
-  #     - name: Install requirements for version tool
-  #       if: ${{ github.event_name == 'pull_request' }}
-  #       run: pip install -r python/requirements.txt
+  package_and_publish:
+    name: Package Charts, Build Docker Image and publish them
+    needs:
+      - tests_passed
+      - select_helm_repo
+    runs-on: ubuntu-latest
+    env:
+      NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+      HELM_REPO: ${{ needs.select_helm_repo.outputs.helm_repository }}
+    if: needs.select_helm_repo.outputs.helm_repository != 'skip'
+    outputs:
+      IMAGE_TAG: ${{ steps.printtag.outputs.IMAGE_TAG }}
+    steps:
+      - name: Install host dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install protobuf-compiler krb5-user libkrb5-dev libclang-dev liblzma-dev libssl-dev pkg-config
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435 # v4
+        if: ${{ github.event_name == 'pull_request' }}
+      - uses: dtolnay/rust-toolchain@1.68.2
+        with:
+          components: rustfmt
+      - name: Install requirements for version tool
+        if: ${{ github.event_name == 'pull_request' }}
+        run: pip install -r python/requirements.txt
 
-  #       # This step checks if the current run was triggered by a push to a pr (or a pr being created).
-  #       # If this is the case it changes the version of this project in all Cargo.toml files to include the suffix
-  #       # "-pr<prnumber>" so that the published artifacts can be linked to this PR.
-  #     - name: Update version if PR
-  #       if: ${{ github.event_name == 'pull_request' }}
-  #       run: python/cargo_version.py -m pr${{ github.event.pull_request.number }}
+        # This step checks if the current run was triggered by a push to a pr (or a pr being created).
+        # If this is the case it changes the version of this project in all Cargo.toml files to include the suffix
+        # "-pr<prnumber>" so that the published artifacts can be linked to this PR.
+      - name: Update version if PR
+        if: ${{ github.event_name == 'pull_request' }}
+        run: python/cargo_version.py -m pr${{ github.event.pull_request.number }}
```

Comment:
This will fail, right? The referenced file doesn't seem to be included in this PR.

Reply:
Ah good catch.

Reply:
e5da0747ff7f4f5e34713955aca5d2ad224fc7b7